File: CS/CustomColorSchemes/Form1.cs
using DevExpress.XtraReports.Wizards;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CustomColorSchemes
{
    public partial class Form1 : Form
    {
        public Form1() {
            InitializeComponent();

            //Register the wizard customization service.
            reportDesigner1.AddService(typeof(IWizardCustomizationService), new WizardCustomizationService());
        }
    }
}

File: VB/CustomColorSchemes/Form1.vb
Imports DevExpress.XtraReports.Wizards
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace CustomColorSchemes
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()

            'Register the wizard customization service.
            reportDesigner1.AddService(GetType(IWizardCustomizationService), New WizardCustomizationService())
        End Sub
    End Class
End Namespace
